lib/puppet/functions/apache/authz_core_config.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+Puppet::Functions.create_function(:'apache::authz_core_config') do
```

Comment:
I'm not sure how to properly document a function with puppet-strings, buy maybe add a sentence or two why the function exists and what the purpose is?

Reply:
sure

Reply:
done!!